Bio/Phylo/CDAOIO.py

```diff
@@ -19,13 +19,15 @@
 them to a file.
 """
 
+import os
+
 from Bio._py3k import StringIO
 
 from Bio import MissingPythonDependencyError
 
 from Bio.Phylo import CDAO
-from ._cdao_owl import cdao_elements, cdao_namespaces, resolve_uri
-import os
+from ._cdao_owl import cdao_namespaces, resolve_uri  # , cdao_elements
```

Comment:
The apparently extra import looks worth double checking - @bendmorris do you recall why it was done? Is this something useful for users of the public API?

Reply:
I don't recall - possibly it was used in an earlier iteration. If it's unused now no reason to keep it there.